README.md

```diff
@@ -14,6 +14,9 @@ Read [CONTRIBUTING](CONTRIBUTING.md) for build and test instructions.
 * `macvlan`: Creates a new MAC address, forwards all traffic to that to the container.
 * `ptp`: Creates a veth pair.
 * `vlan`: Allocates a vlan device.
+#### Windows: windows specific
+* `l2bridge`: Creates a bridge, adds the host and the container to it.
```

Comment:
The Windows L2 bridge really isn't an L2 bridge, since the HNS bits do a lot of stuff with IP and DNS because of how HNS works.  We'd like the plugin to be renamed to "win-bridge" or something like that instead, since "l2bridge" is somewhat of a mis-nomer give it does things from L2 all the way to L4 or higher.